tests/present_and_fetch.rs

```diff
@@ -1,3 +1,5 @@
+// TODO: Once winit is updated again, restore this test.
```

Comment:
Winit is released, so presumably this can be updated. Though https://github.com/notgull/winit-test will need a release.

Reply:
https://github.com/notgull/winit-test/pull/8